src/CMakeLists.txt

```diff
@@ -48,10 +48,16 @@ endif()
 
 
 #Link the required dependencies
-target_link_libraries(${PROJECT_NAME}
-	${IIO_LIBRARIES}
-        ${LOG4CPP_LIBRARies}
-)
+if (ENABLE_LOG)
+	target_link_libraries(${PROJECT_NAME}
```

Comment:
I think you can do the usual target_link_libraries with IIO as that is required and if ENABLE_LOG just add glog
```cmake
target_link_libraries(${PROJECT_NAME} ${IIO_LIBRARIES})

if (ENABLE_LOG) 
       target_link_libraries(${PROJECT_NAME} glog::glog)
endif()
```

Reply:
Modified.